docs: Update limits_and_triggers.adoc with new block limits (#1188)
starknet-io/starknet-docs: components/Starknet/modules/tools/pages/limits_and_triggers.adoc

[%autowidth.stretch]
|===
|Entity | Description | Sepolia | Goerli (deprecated) | Mainnet

> |Block time | If no other limit closes a block, the pending block is closed.| 4 minutes |4 minutes |4 minutes

Reviewer: ```suggestion
|Block time | The maximum amount of time within which a pending block is closed, if no other limit is met. | 4 minutes |4 minutes |4 minutes
```

> |Block limit (Cairo steps)|Starknet enforces a limit on the number of steps that can be completed

Reviewer: ```suggestion
|Block limit (Cairo steps)| The maximum number of Cairo steps that can be completed
```
